tests/app/app_context_test_case.py

```diff
@@ -11,19 +16,56 @@ class AppContextTestCase(unittest.TestCase):
     LOGIN_DISABLED = False
 
     def setUp(self):
+        self._ddb = mock_dynamodb2()
+        self._ddb.start()
+
         setting_overrides = {
             'SQLALCHEMY_DATABASE_URI': 'sqlite://',
             'LOGIN_DISABLED': self.LOGIN_DISABLED,
-            'EQ_DYNAMODB_ENABLED': False
+            'EQ_DYNAMODB_ENDPOINT': None,
```

Comment:
Why?

Reply:
moto doesn't mock dynamo if an endpoint is specified